test/unit/migrations/037-test.js

```diff
@@ -0,0 +1,129 @@
+const assert = require('assert')
+let id = 0
+const migrationTemplate = require('../../../app/scripts/migrations/036')
```

Comment:
Should this be the template file?

Reply:
Or should this be 37?

```suggestion
const migration37 = require('../../../app/scripts/migrations/037')
```